internal/server/iigointernal/executive_test.go

```diff
@@ -690,8 +690,10 @@ func TestReplyAllocationRequest(t *testing.T) {
 		},
 	}
 
-	rules.PullRuleIntoPlay("allocation_decision")
-	rules.PullRuleIntoPlay("check_allocation_rule")
+	rulesInPlay := map[string]rules.RuleMatrix{}
+
+	rules.PullRuleIntoPlayInternal("allocation_decision", generateRuleStore(), rulesInPlay)
```

Comment:
even in a test these should be checked

Reply:
Oooo nice catch, changed in latest commit